### PR TITLE
hide ref_id from export

### LINKED ISSUE #1: [FEATURE] Hide ref_id column from exported spreadsheet
**Is your feature request related to a problem? Please describe.**
Ref_ids carry a fair bit of meaning here, so we want to tightly control how they can be minted and updated.  To do so, we have a [plugin](https://github.com/Smithsonian/caas_aspace_refid) that uses established logic to autogenerate new archival object refids, but there are still other means by which users can intentionally or unintentionally create or change refids in non-standard ways.  This bulk update workflow is one of those ways.

**Describe the solution you'd like**
So that untracked ref_ids can't make their way into ASpace via the bulk update workflow, we've opted to hide the ref_id column from the original spreadsheet export (this tracks with the custom templates that we already use for the "Load via Spreadsheet" workflow).

**Describe alternatives you've considered**
See https://github.com/Smithsonian/caas_aspace_refid/issues/15

**Additional context**
In the future, this forked version of the plugin will be unecessary as this functionality is being incorporated into aspace 4.0.  When that is complete, we will roll out a lighter weight plugin that accomplishes the same by hiding the ref_id column as a selectable field for spreadsheet export.

### PR DESCRIPTION
**Pull request by lorawoodford**
Hides the "ref_id" option from the download spreadsheet form to address: https://github.com/Smithsonian/caas_aspace_refid/issues/15

Also adds some basic pr and issue templates to maintain consistency across our repos.

Closes #1

### CODE REVIEW

**Comment by lorawoodford** [comment]
@crugas Would you be willing to look at this just to follow the "another set of eyes has looked before merging" standard?  This is extremely straightforward, and involves us (temporarily) forking this older plugin in order to remove the ability for folks to export and, as a consequence, import new/changed/untracked ref_ids via the bulk update spreadsheet workflow.  Once we adopt v4+ this forked repo will go away and we'll instead use a similar plugin to again hide this column from export.  This is one minor step we decided to take to attempt to thwart some of the many ways folks could unintentionally alter ref_ids in a way that the ref_id generation plugin and db table doesn't know about.

Once "approved" I'll get this up on aspace02.

**Review by crugas — APPROVED** [review]
Looks good to me! I'm curious if removing the ref_id lines in the code for the plugin affect the background job on re-import. Such as if there is any code within the background job that looks specifically for the ref_id field even if it isn't displayed in the spreadsheet or download form in the SUI. I know you can hide columns in Excel from view, but those columns still exist in Excel so re-imports shouldn't be affected.

**Comment by lorawoodford** [comment]
@crugas Nope, if someone went in and added a (properly formatted/properly headed) `ref_id` column into that spreadsheet and then added a new/changed value for the ref_id in that column, the plugin would still accept it and update the ref_id during the upload process.  This is just us taking a gamble and assuming that folks won't hack their spreadsheets like that!  If the column and column data are entirely missing, however, the plugin does nothing and the existing values are persisted/not updated (which is what we want).

**Comment by crugas** [comment]
@lorawoodford Awesome! No spreadsheet hackers allowed!

**Pull request merged**
